pkg/resources/envoy/service.go

```diff
@@ -85,10 +85,20 @@ func getExposedServicePorts(extListener v1beta1.ExternalListenerConfig, brokersI
 		}
 	}
 
+	// append anycast port
 	exposedPorts = append(exposedPorts, corev1.ServicePort{
 		Name:       fmt.Sprintf(kafkautils.AllBrokerServiceTemplate, "tcp"),
 		TargetPort: intstr.FromInt(int(extListener.GetAnyCastPort())),
 		Port:       extListener.GetAnyCastPort(),
+		Protocol:   corev1.ProtocolTCP,
+	})
+
+	// append envoy admin port
+	exposedPorts = append(exposedPorts, corev1.ServicePort{
+		Name:       fmt.Sprintf("envoy-admin"),
+		TargetPort: intstr.FromInt(9901),
```

Comment:
Please use the ingressConfig.EnvoyConfig.GetEnvoyAdminPort here otherwise it is baked in.

Reply:
good catch @baluchicken - thanks, apparently I messed up the PR commit when pushing.
Fixed this - please have another look